src/client/api/api.js

```diff
@@ -2,7 +2,7 @@ import axios from "axios";
 
 const api = axios.create({
   baseURL: "http://192.168.2.102:5001/api",
-  timeout: 5000,
+  timeout: 50000,
```

Comment:
model would take more than 5 secs to parse image so just changed to 50 secs for now